lib/src/request.dart

```diff
@@ -130,11 +131,43 @@ class Request extends Message {
       {Map<String, String> headers, Map<String, Object> context})
       : this('DELETE', url, headers: headers, context: context);
 
-  Request._(this.method, this.url,
-      body,
-      Encoding encoding,
+  /// Creates a new `multipart/form-data` [Request] to [url], which can be a
```

Comment:
Consider linking `multipart/form-data` to an explanation of the format (maybe Wikipedia).